content/influxdb/cloud-iox/union.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The UNION clause
+list_title: The UNION clause
+description: > 
+    Use the `UNION` clause to combine the results of two or more queries into a single set of results.
+menu:
+  influxdb_cloud_iox:
+    name: The UNION clause
+    parent: Explore data using SQL
+weight: 295
+---
+
+The `UNION` clause combines the results of two or more `SELECT` statements into a single results set.
+
+Key points to keep in mind when using the `UNION` clause:
+
+ - the number of columns in the column list must be the same
+ - the columns in the column list must be in the same order
+ - the data types in the column list must be the same or compatible  
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+## Syntax
+
+```sql
+SELECT a, b, c
+FROM measurement_1
+UNION [ALL]
+SELECT a, b, c
+FROM measurement_2
+``` 
+
+## Examples
+
+Basic `UNION` query:
+
+```sql
+SELECT "pH" as "water_pH", "time", "location"
+FROM "h2o_pH"
+UNION 
+SELECT "location", "time", "randtag" 
+FROM "h2o_quality"
+```
+Results:
+
+| location     | time                     | water_ph |
+| :----------- | :----------------------- | :------- |
+| coyote_creek | 2019-08-21T00:00:00.000Z | 8.0      |
+| coyote_creek | 2019-08-21T00:18:00.000Z | 7.0      |
+| coyote_creek | 2019-08-21T00:42:00.000Z | 8.0      |
+
+The query returns the location, time and pH from the 2 tables.
+
+`UNION ALL` query using an `OVER` clause:
+
+```sql
+SELECT 'bottom' as type, time, water_level FROM
+  (SELECT time, "water_level", row_number() OVER (order by water_level) as rn FROM h2o_feet) where rn <= 3 
+UNION ALL 
+SELECT 'top' as type, time, water_level FROM 
+  (SELECT time, "water_level", row_number() OVER (order by water_level DESC) as rn FROM h2o_feet) where rn <= 3
+  ```
+Results:
+| time                     | type   | water_level |
+| :----------------------- | :----- | ----------- |
+| 2019-08-28T07:24:00.000Z | top    | 9.964       |
+| 2019-08-28T07:18:00.000Z | top    | 9.957       |
+| 2019-08-28T07:30:00.000Z | top    | 9.954       |
+| 2019-08-28T14:30:00.000Z | bottom | -0.61       |
+| 2019-08-29T15:18:00.000Z | bottom | -0.594      |
+| 2019-08-28T14:36:00.000Z | bottom | -0.591      |
+
+The query returns the type of `water_level`, top or bottom, using the `OVER` clause to window the data to get the top and bottom 3 results.
```

Comment:
Move the explanation above the query. I think we probably need to explain more of what's happening in the query.

```suggestion
```